app.lith.Lith.json

```
@@ -0,0 +1,44 @@
+{
+    "id": "app.lith.Lith",
+    "runtime": "org.kde.Platform",
+    "runtime-version": "5.15",
+    "sdk": "org.kde.Sdk",
+    "command": "Lith",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=network",
+        "--share=ipc",
+        "--talk-name=org.freedesktop.Notifications",
+        "--talk-name=org.kde.StatusNotifierWatcher",
+        "--talk-name=com.canonical.indicator.application",
+        "--talk-name=org.ayatana.indicator.application",
+        "--talk-name=org.freedesktop.portal.Fcitx",
+        "--own-name=org.kde.StatusNotifierItem-1-1",
+        "--own-name=org.kde.StatusNotifierItem-2-1",
+        "--own-name=org.kde.StatusNotifierItem-3-1",
+        "--own-name=org.kde.StatusNotifierItem-4-1",
+        "--own-name=org.kde.StatusNotifierItem-5-1",
+        "--own-name=org.kde.StatusNotifierItem-6-1",
+        "--own-name=org.kde.StatusNotifierItem-7-1",
+        "--own-name=org.kde.StatusNotifierItem-8-1",
+        "--own-name=org.kde.StatusNotifierItem-9-1"
```

```suggestion
        "--own-name=org.kde.*",
```

I know yours is kinda better security wise but it will still break if 2 flatpak apps use the same PID inside the sandbox.

---

Are you sure this works on older versions of Flatpak? I haven't actually tested this right now but I remember spending a considerable amount of time before I found a variant of this that works at least somehow with Ubuntu 16.04.
Anyway, I'm merging this, 16.04 is really old but I'm curious if it actually helps.